### PR TITLE
[WebRTC] Add new APIs for MediaNullSource and change MediaFileSource ctor

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

Add new APIs for MediaNullSource
Change MediaFileSource ctor

### API Changes ###

 - TCSACR-510

### CODE REVIEW

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs [hunk 1]
```diff
@@ -160,11 +206,14 @@ public TransceiverCodec TransceiverCodec
         /// Retrieves the supported transceiver codecs.
         /// </summary>
         /// <remarks>
-        /// This API is not supported in <see cref="MediaFileSource"/>, <see cref="MediaPacketSource"/>.
+        /// This API is not supported in <see cref="MediaFileSource"/>, <see cref="MediaPacketSource"/>.<br/>
+        /// If user want to get supported codecs for each audio or video in<see cref="MediaNullSource"/>,
```
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs:210]: please put a space between "video in" and `<see`

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaFileSource.cs [hunk 2]
```diff
@@ -82,6 +81,98 @@ public bool IsLooping
...
+            WebRtc.ValidateWebRTCState(WebRTCState.Idle);
+
+            NativeWebRTC.SetTransceiverDirection(WebRtc.Handle, SourceId.Value, type, direction).
+                ThrowIfFailed("Failed to set transceiver direction");
+        }
+
+        /// <summary>
+        /// Gets the pause status of media file source.
+        /// </summary>
+        /// <param name="type">The media type.</param>
```
[the remark this answers is not in the document]
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/MediaFileSource.cs:136]: Added.

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaNullSource.cs [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+using System;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using NativeWebRTC = Interop.NativeWebRTC;
+
+namespace Tizen.Multimedia.Remoting
+{
+    /// <summary>Represents a null source.</summary>
+    /// <remarks>
+    /// If WebRTC add this source, it only receives media stream.<br/>
```
[the remark this answers is not in the document]
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/MediaNullSource.cs:26]: I changed it like below.
`If you add this source, WebRTC only receives media stream.`

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaNullSource.cs [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+            NativeWebRTC.SetTransceiverCodec(WebRtc.Handle, SourceId.Value, type, codec).
+                ThrowIfFailed("Failed to set transceiver codec");
+        }
+
+        /// <summary>
+        /// Retrieves the supported transceiver codecs.
+        /// </summary>
+        /// <remarks>
+        /// This API is not supported in <see cref="MediaFileSource"/>, <see cref="MediaPacketSource"/>.
```
[the remark this answers is not in the document]
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/MediaNullSource.cs:97]: This is mistake. I'll remove it.

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs (hunk 1 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs:210]: Fixed

**Review by hancepark — APPROVED** [review]

**Pull request merged**